superset-frontend/plugins/plugin-chart-cartodiagram/README.md

```diff
@@ -1,50 +1,61 @@
-# plugin-chart-cartodiagram
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
 
-An OpenLayers map that displays charts for single features. Superset Chart Plugin.
+  http://www.apache.org/licenses/LICENSE-2.0
 
-## Usage
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
 
-To build the plugin, run the following commands:
+## @superset-ui/plugin-chart-cartodiagram
 
-```
-npm ci
-npm run build
-```
+[![Version](https://img.shields.io/npm/v/@superset-ui/plugin-chart-cartodiagram.svg?style=flat-square)](https://www.npmjs.com/package/@superset-ui/plugin-chart-cartodiagram)
 
-Alternatively, to run the plugin in development mode (=rebuilding whenever changes are made), start the dev server with the following command:
+This plugin provides ECharts viz on an OpenLayers map for Superset:
 
-```
-npm run dev
-```
+- Timeseries Chart (combined line, area bar, scatter, smooth, step)
+- Pie Chart
```

Comment:
mixes time series is missing

Reply:
I understand the Timeseries combined line as such.